Formula/s/seqan3.rb

```diff
@@ -0,0 +1,51 @@
+class Seqan3 < Formula
+  desc "Modern C++ library for sequence analysis"
+  homepage "https://www.seqan.de"
+  url "https://github.com/seqan/seqan3/archive/refs/tags/3.4.0.tar.gz"
+  sha256 "8e000e6788f1e2ada071b36f64231d56f18e2d687ab4122d86cd3aefc6c87743"
+  license all_of: ["BSD-3-Clause", "CC-BY-4.0", "CC0-1.0", "MIT", "Zlib"]
+
+  depends_on "cereal" => :build
+  depends_on "cmake" => :build
+
+  # https://github.com/seqan/seqan3?tab=readme-ov-file#dependencies
+  fails_with :clang do
+    build 1699
+    cause "needs Clang 17 or newer"
+  end
+
+  # https://github.com/seqan/seqan3?tab=readme-ov-file#dependencies
+  fails_with :gcc do
+    version "11"
+    cause "needs GCC 12 or newer"
+  end
+
+  resource "cmake-scripts" do
+    url "https://github.com/seqan/cmake-scripts.git", revision: "d2a54ef555b6fc2d496a4c9506dbeb7cf899ce37"
+  end
```

Comment:
maybe spread the line?

is this always the latest (if so maybe worth adding a code comment)
```suggestion
  resource "cmake-scripts" do
    url "https://github.com/seqan/cmake-scripts.git", 
        revision: "d2a54ef555b6fc2d496a4c9506dbeb7cf899ce37"
  end
```

Reply:
From what I can tell, there's only been one version - the current one (the repo only has [five commits](https://github.com/seqan/cmake-scripts/commits/main/)). Another option would be to remove it entirely:

```ruby
inreplace "cmake/seqan3-package.cmake", "CPMGetPackage (use_ccache)", ""
```